shared-modules/maturin/fedc.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
`shared-modules` as a directory is usually used for the `shared-modules` as a git submodule. So I don't recommend using that directory name to organise things in a flathub project.

Reply:
Move them to `includes`.